http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3037,33 +3117,33 @@ components:
       description: ""
       required:
         - id
-        - googleProject
+        - cloudContext
         - zone
         - name
-        - googleId
+        - serviceAccount
         - samResourceId
         - status
         - auditInfo
         - size
         - diskType
         - blockSize
         - labels
+        - formattedBy
       properties:
         id:
           type: integer
-          description: Internal Leonardo ID of the persistent disk
-        googleProject:
-          type: string
-          description: The google project used to create the persistent disk
+          description: The user-supplied ID of the persistent disk
```

Comment:
nit: I believe the IDs being used here for both Azure and GCP are "Cloud-platform provided"
"user-supplied" while correct is a bit vague, and could mean the user decides what this is.